README.md

```diff
@@ -119,6 +131,48 @@ With just a click of a button, you can navigate backwards and forwards through t
   <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/revision-navigation.gif" alt="Revision Navigation" />
 </figure>
 
+# Accelerate PR Reviews
+
+PR Reviews can take time and require context switching between GitHub, email, and your IDE. Launchpad consolidates PRs into an actionable hub where you can manage them all in one place. Seamlessly prioritize tasks and identify bottlenecks - all within VS Code when you integrate your Github or host provider. Work on multiple branches simultaneously with Worktrees — hotfixes, new features, or experiments without disrupting your main workspace.
+
+## Launchpad `Pro`
+
+Launchpad brings all of your GitHub pull requests into a unified, actionable list to better track work in progress, pending work, reviews, and more. Stay focused and take action on the most important items to keep your team unblocked. [Learn more](https://gitkraken.com/solutions/launchpad?utm_source=gitlens-extension&utm_medium=in-app-links)
+
+<figure align="center">
+  <img src="https://raw.githubusercontent.com/gitkraken/vscode-gitlens/main/images/docs/launchpad.png" alt="Launchpad" />
+</figure>
+
+## Worktrees `Pro`
+
+Efficiently multitask by minimizing the context switching between branches, allowing you to easily work on different branches of a repository simultaneously.
```

Comment:
@eatmorespinach worktrees still require context switching (you have to switch between them to do work), but the main benefit is that they allow you to switch contexts without interrupting your because you don't need to stash/commit current changes or leave your current branch.

```suggestion
Worktrees enable efficient multitasking by allowing you to work on multiple branches without stashing changes or leaving your current branch. They preserve your workflow while letting you shift focus when needed. For example, you can easily review a pull request on a worktree in a separate VS Code window with GitLens.
```